<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FibresViewControls</class>
 <widget class="QWidget" name="FibresViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="btn1_preproc">
       <property name="text">
        <string>Preprocessing</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn4_fibres">
       <property name="text">
        <string>Fibres</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn3_lapsolves">
       <property name="text">
        <string>Laplace Solves</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn2_extractsurf">
       <property name="text">
        <string>Extract Surfaces</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QVTKOpenGLWidget" name="widget_1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnY_visualise">
       <property name="text">
        <string>Visualise</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnX_cancel">
       <property name="maximumSize">
        <size>
         <width>119</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
